examples/mliap/README

```diff
@@ -109,4 +109,70 @@ in.mliap.nn.cu
 -------------------------
 Run a neural network potential for Cu, a combination of SNAP descriptors and the "nn" model style
 
+in.mliap.unified.lj.Ar
+-----------------------
+This reproduces the output of examples/melt/in.melt,
+but using the ``unified`` keyword and
+the Python coupling to PyTorch.
+
+This example can be run in two different ways:
+
+1: Running a LAMMPS executable: in.mliap.unified.lj.Ar
+
+You can run the example as follows
+
+`lmp -in in.mliap.unified.lj.Ar -echo both`
```

Comment:
FYI the input now includes the python call to create the pickle via the LAMMPS shell command.

Reply:
I've refactored this some more. Now it is using the LAMMPS python command so no external file is needed.